<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatSearch</class>
 <widget class="QWidget" name="ChatSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;聊天记录搜索&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索内容：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>721</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>sender</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>receiver</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>message</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>timestamp</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>导出到文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除该记录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
